### PR TITLE
Update footer text of checkout pages.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
Before: 
picture 215

After: 
picture 216

### CODE REVIEW

File: app/views/fragments/global/footer.scala.html [hunk 1]
```diff
@@ -16,14 +16,15 @@
...
+              By proceeding, you are agreeing to our Terms and Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.
               For help with Guardian and Observer subscription services please email
                   <a href="mailto:@email">@email</a> or call @phone
             </p>
             <p>
                 You may also find help in our
                 <a href="@maybeProduct.getOrElse(Digipack).faqHref"
                    title="Frequently Asked Questions">
-                   Frequently Asked Questions.
+                   Frequently Asked Questions and in the Digital Pack terms and conditions.
```
Comment (walaura) [inline app/views/fragments/global/footer.scala.html]: should this be all the same link?

Comment (LATaylor-guardian) [reply app/views/fragments/global/footer.scala.html]: I just noticed this - have updated.

**Comment by lmath** [comment]
Are there links available for the privacy policy and the digital pack T&Cs?

**Comment by LATaylor-guardian** [comment]
@lmath You'd know better than me - can you point me to them?

**Pull request closed without merging**